### ISSUE TITLE
Dashboard page doesn't have a heading level 1

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#12]
URL: https://hcdev.openplus.ca/en/user/197
**User Agent:** Chrome/FF
Language: both

**Issue description:** The page does not have heading <h1> . Headings should be in hierarchical order.
**Steps to reproduce**: source code
**Expected result:** All pages should have H1 and this page seems like "Recalls and safety alerts" is the name of the application and each page should have an H1 in the main content.

**Comment by joel-osc** [comment]
Fixed.

**Comment by mmiron-HC** [comment]
@joel-osc  please don't close the issues. We need to retest these and we will close them. just reassign it to the person who raised the ticket.

**Comment by ndolter** [comment]
Although an 'H1' is now present, there shouldn't be 2 instances of it.  Currently, both 'Dashboard' and 'UAT testing' are listed as 'H1'.  UAT testing should instead be 'H2' so as to be in a hierarchical order.
![image](https://user-images.githubusercontent.com/68117281/135904019-49c5bf7c-6fac-40ad-978a-0c7bfbd21a55.png)

**Comment by davidgotlieb** [comment]
Hi @ndolter , The UAT header is just temporary as we needed to add a quick link for our testing by programs. That will be deleted shortly so this shouldn't be an issue. It isn't permanently on the page.

**Comment by ndolter** [comment]
Closing ticket since the 2nd 'H1' has now been removed.
